main.go

```diff
@@ -81,14 +82,14 @@ func main() {
 		},
 		&cli.StringFlag{
 			Name:        "srv-record-target",
-			Usage:       "Target of the RFC2782 SRV Record. Usualy something resembling your hostname.",
+			Usage:       "Target of the RFC2782 SRV Record. Usually something resembling your hostname.",
```

Comment:
suggestion: This should indicate the desired format, namely the trailing dot to help users holding it right.

Reply:
Makes sense, added in ee66aca68b3f5bcff27967c3b20e9a7864a099aa.